PROCESSORS.md

```diff
@@ -177,53 +177,43 @@ def dump_to_sql(tables,
 - `updated_id_column` - Optional name of a column that will be added to the output data containing the id of the updated row in DB.
 
 
-#### cache
-Cache results from running a series of steps, if cache exists - loads from cache instead of running the steps.
+#### checkpoint
 
-Cache invalidation should be handled manually - by deleting the cache path.
+Save results from running a series of steps, if checkpoint exists - loads from checkpoint instead of running the steps.
+
+Checkpoint invalidation should be handled manually - by deleting the checkpoint path (by default at `.checkpoints/checkpoint_name`)
 
 ```python
-def cache(steps*, cache_path):
+def checkpoint(checkpoint_name, checkpoint_path='.checkpoints', steps=None, resources=None):
     pass
 ```
 
-- `steps*` - step functions to run to create the cache (same as the `Flow` class arguments)
-- `cache_path` - path to a unique directory that will hold the cache for the provided series of steps
+- `checkpoint_name` - The checkpoint is saved to a datapackage at `cache_path/checkpoint_name`
+- `cache_path` - Relative or absolute path to save checkpoints at, default to relative path `.checkpoints`
+- `steps` - Iterable of steps to checkpoint, when not provided, uses the previous steps in the flow
+- `resources` - Limit the checkpointing only to specific resources, same semantics as `load` processor `resources` argument
+
+To improve code readiblity, the checkpoint processor can be place anywhere in the flow and it will checkpoint the previous steps:
```

Comment:
```suggestion
To improve code readability, the checkpoint processor can be placed anywhere in the flow and it will checkpoint the previous steps:
```